### PR TITLE
Upgrade ssb-refs, update use of refs.FeedRef

### PR DESCRIPTION
**Pull request by cblgh**
cc @cryptix; had to tweak some stuff in netsim after upgrading ssb-refs to ssb-refs@ssb-uri

lmk if i messed any part up!

### CODE REVIEW

File: internal/keys/keys.go [hunk 1]
```diff
@@ -53,11 +53,12 @@ func NewKeyPair(r io.Reader) (*KeyPair, error) {
 		return nil, fmt.Errorf("ssb: error building key pair: %w", err)
 	}
 
+	feed, err := refs.NewFeedRefFromBytes(kp.Public[:], refs.RefAlgoFeedSSB1)
```
Comment (cryptix) [inline internal/keys/keys.go:56]: 👌🏻

File: sim/commands.go [hunk 2]
```diff
@@ -355,7 +355,7 @@ func queryIsFollowing(srcPuppet, dstPuppet *Puppet) (bool, error) {
 	arg := struct {
 		Source *refs.FeedRef `json:"source"`
 		Dest   *refs.FeedRef `json:"dest"`
```
Comment (cryptix) [inline sim/commands.go:357]: could also remove the `*` here (usually just need pointers for optionals in JSON)

**Review by cryptix — APPROVED** [review]
LGTM

File: sim/commands.go (hunk 2 above)
Comment (cblgh) [reply sim/commands.go:357]: good to know! think i'll just keep em as is for now haha

**Pull request merged**